api/gobgp.proto

```diff
@@ -265,6 +265,8 @@ message ListPathRequest {
   // enable_only_binary == true means that only nlri_binary and pattrs_binary
   // will be used instead of nlri and pattrs for each Path in ListPathResponse.
   bool enable_only_binary = 9;
+  // max ammount of paths to be allocated, default is 16K
+  int64 batch_size = 10;
```

Comment:
Why not unsigned type like uint64?

Reply:
just to avoid sign bit while conversion to int on x64, at the other hand with uint64 error return will be unnecessary